plugins/commands/create.js

```diff
@@ -94,14 +128,18 @@ module.exports = () => ({
             const data = request.payload;
             let commandSpec;
             let commandBin;
+            let multipartCheckResult = { valid: false };
 
             // if Content-type is multipart/form-data, both command binary and meta are posted
             if (request.headers['content-type'].startsWith('multipart/form-data')) {
-                if (data.binary === undefined || data.spec === undefined) {
-                    return reply(boom.badRequest('Posted with invalid form.'));
+                multipartCheckResult = checkValidMultipartPayload(data);
+
+                if (multipartCheckResult.valid) {
+                    commandSpec = data.spec;
+                    commandBin = data.binary;
```

Comment:
I don't want to add `if` here to get `habitat` binary, so I change the key of data in [this PR](https://github.com/screwdriver-cd/sd-cmd/pull/26).

Reply:
I think `binary` is a little confusing. It's similar to a type of command format. How about `file` or `bin`?

Reply:
To see [specification](https://github.com/screwdriver-cd/screwdriver/blob/master/design/commands.md#specification), `file` is used for specified it. I `file` is nice.